sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample16_CrossReceiverMessageSettlement.md

```diff
@@ -0,0 +1,68 @@
+# Cross Receiver Message Settlement
+
+The message settlement APIs on the `ServiceBusReceiver` require passing in the
+`ServiceBusReceivedMessage`. This can be limiting for scenarios in which the message needs to be persisted and then
+settled across process boundaries. There are two strategies that can be used for settling a message with a different
+receiver. The correct strategy to use will depend on the specific scenario.
```

Comment:
```suggestion
receiver. The recommended strategy depends on the specific application scenario.
```
My intent here was to preserve what you were saying but soften the tone just a bit.